test_api.py

```diff
@@ -2869,43 +2899,50 @@ def test_get_system_profile_ignores_culled(self):
 
 
 class QueryStalenessGetHostsIgnoresCulledTestCase(QueryStalenessGetHostsBaseTestCase, DeleteHostsBaseTestCase):
-    def test_patch_ignores_culled(self):
+    @patch("api.host.emit_patch_event")
```

Comment:
To be consistent with the [_delete_ tests](https://github.com/RedHatInsights/insights-host-inventory/blob/a8c07151bd386b8d1c90bddafa13a42b55df3c71/test_api.py#L2939), the very producing function (_tasks.emit_event_) should be mocked. Mocking _api.host.emit_patch_event_ would be analogical to mocking _lib.host_delete._emit_event_.

```suggestion
    @patch("api.host.emit_event")
```